src/demo/index.jsx

```diff
@@ -10,13 +10,12 @@ $(document).ready(() => {
   canvas.width = backgroundCanvas.width = constants.canvasWidth;
   canvas.height = backgroundCanvas.height = constants.canvasHeight;
 
-  const state = setState({
+  setState({
     currentMode: Modes.Training,
     canvas,
     backgroundCanvas,
     uiContainer: document.getElementById('ui-container')
```

Comment:
As we discussed before, it seems we could take the canvases and the UI element out of the state at some point, if we define this state as "everything we need to persist to restore the previous state upon reload".